assignment/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java

```diff
@@ -3394,7 +3399,12 @@ protected String build_instructor_grade_submission_context(VelocityPortlet portl
             context.put("value_grade", (String) state.getAttribute(GRADE_SUBMISSION_GRADE));
         } else {
             // format to show one decimal place in grade
-            context.put("value_grade", assignmentService.getGradeDisplay((String) state.getAttribute(GRADE_SUBMISSION_GRADE), gradeType, assignment.isPresent() ? assignment.get().getScaleFactor() : null));
+            if (assignment.isPresent() && assignment.get().getTypeOfGrade() == SCORE_GRADE_TYPE) {
```

Comment:
AssignmentServiceImpl.getGradeDisplay is able to handle displaying any grade type there is no need to perform this check here